recipes/process-main-module/workflow.yaml

```
@@ -0,0 +1,24 @@
+version: "1"
+nodes:
```

```suggestion

nodes:
```

let's add little bit of space to make it prettier